modules/avatar/react/lib/Avatar.tsx

```diff
@@ -81,6 +85,9 @@ const StyledContainer = styled('button', {
   }),
   ({onClick}) => ({
     cursor: onClick ? 'pointer' : 'default',
+  }),
+  ({imageLoaded, url}) => ({
+    backgroundColor: !url || imageLoaded ? 'transparent' : colors.soap200,
```

Comment:
We don't want to transition `backgroundColor`. That requires a repaint in the browser. We want to transition `opacity` instead:

* `backgroundColor`: https://csstriggers.com/background-color
* `opacity`: https://csstriggers.com/opacity